jobs/iptables-manager/templates/bin/iptables-manager_ctl.erb

```diff
@@ -21,9 +21,12 @@ exec 2>> $LOG_DIR/$JOB_NAME.stderr.log
 
 case $1 in
   start)
-    iptables -F #Flush all existing rules
```

Comment:
why was this `iptables --flush` step added earlier and removed now?

And since, it is getting removed, I am assuming, executing an IpTable rule twice (even if it exists) doesn't throw an error?

Reply:
fat finger. Removed it by mistake. Added it back.

Reply:
thanks!